.github/workflows/pkgprnew.yaml

```diff
@@ -0,0 +1,31 @@
+name: pkg.pr.new
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
In the same way of limiting permissions of this, I think this should have the `persist-credentials: false` option added